### DISCUSSION TITLE
How to start the development server after completing local setup?

### DISCUSSION THREAD

**Discussion by prashant-gurung899** [spree/spree#12247] [Q&A]
I following the instructions from this doc to do the local setup.

**Comment by prashant-gurung899** [comment]
I followed the instructions from the following doc to do the local setup:
https://spreecommerce.org/docs/developer/contributing/developing-spree

**Reply by damianlegawiec** [reply]
Hi  @prashant-gurung899 

Do you want to start a development server to develop spree itself or you're just starting a new project based on spree?

If the first, you need to follow the instructions from the guide you mentioned, and later go to `sandbox` directory and run standard rails server - `bin/rails server` - https://guides.rubyonrails.org/getting_started.html#starting-up-the-web-server

**Reply by prashant-gurung899** [reply]
@damianlegawiec I want to start a development server so that i can practice e2e tests on it.I want to practice e2e tests on the admin panel locally.
This is what i get by doing `bin/rails server`
![image](https://github.com/user-attachments/assets/8d157123-900b-4b5e-8b93-d57f895af9aa)

**Reply by prashant-gurung899** [reply]
@damianlegawiec The following is setup using the template(https://spreecommerce.org/docs/developer/getting-started/quickstart) and it is what i want but i am unable to do so with the forked one:
![image](https://github.com/user-attachments/assets/9c0d8f61-b3a4-4191-a807-ece98a7bf4d1)
